content/operate/redisinsight/install/configuration.md

```diff
@@ -0,0 +1,161 @@
+---
+categories:
+- docs
+- operate
+- redisinsight
+linkTitle: Configuration settings
+title: Configuration settings
+weight: 5
+---
+To configure Redis Insight with environment variables:
+
+1. Follow the instructions to set environment variables for your operating system:
+
+    - [macOS](https://apple.stackexchange.com/a/106814)
+    - [Windows](https://www.architectryan.com/2018/08/31/how-to-change-environment-variables-on-windows-10/)
+    - [Linux](https://askubuntu.com/a/58828)
+    - [Docker](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file)
+
+1. Set the environment variables.
```

Comment:
Do we need this line?

Reply:
Haha... no. :) I'll remove it next push.